website/profile/utils.py

```diff
@@ -107,6 +111,11 @@ def serialize_contributors(contribs, node, **kwargs):
         for contrib in contribs
     ]
 
+def serialize_visible_contributors(node):
+    # This is optimized when node has .include('contributor__user__guids')
+    return [
+        serialize_user(c, node) for c in node.contributor_set.all() if c.visible
```

Comment:
It seems like` include` is broken if that's the outcome. Without `include` filtering on visible works, right?